### PR TITLE
Feature/doc versions

### PR DESCRIPTION
**Pull request by boueslati**

### CODE REVIEW

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/versions/DiffService.java [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+import org.suigeneris.jrcs.diff.delta.Delta;
+import org.suigeneris.jrcs.util.ToString;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by exo on 10/3/16.
+ */
+public class DiffService {
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/versions/DiffService.java]: It seems you created a new diff service. Did you remove the previous one ?

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/versions/DiffService.java [hunk 2]
```diff
@@ -0,0 +1,275 @@
+package org.exoplatform.ecm.webui.component.explorer.versions;
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/versions/DiffService.java]: PLease add unit tests for this new service.

Comment (boubaker) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/versions/DiffService.java]: @thomasdelhomenie shouldn't we move Wiki Service to commons project and reuse it. WDYT ?

Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/versions/DiffService.java]: Yes, we must avoid duplicating code.

**Pull request closed without merging**